frontends/web/src/routes/account/receive/receive.jsx

```diff
@@ -276,7 +276,8 @@ export default class Receive extends Component {
                     <Status type="warning">
                         {paired === false && t('warning.receivePairing')}
                     </Status>
-                    <Header title={<h2>{t('receive.title')}</h2>} />
+                    {/* <Header title={<h2>{t('receive.title')}</h2>} /> */}
+                    <Header title={<h2>{`Receive ${account.name}`}</h2>} />
```

Comment:
make it `t('receive.title', { accountName: account.name })`, then we can simplify modify it on locize